### PR TITLE
Add Hebrew online courses

### PR DESCRIPTION
**Pull request by tomer**
## What does this PR do?
Add Hebrew courses by [campus.gov.il](https://campus.gov.il)

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [X] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: courses/free-courses-he.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### Index
+
+* [C++](C++)
+* [Python](#python)
+* [R](#R)
+
+
+### C++
```
Comment (LuigiImVector) [inline courses/free-courses-he.md:8]:
```suggestion
### C++

```

File: courses/free-courses-he.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* [מבוא לתכנות בשפת C++](https://campus.gov.il/course/course-v1-basmach-pc264/) (קמפוסIL ובסמ״ח)
+
+
+### Python
```
Comment (LuigiImVector) [inline courses/free-courses-he.md:12]:
```suggestion
### Python

```

File: courses/free-courses-he.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+* [Self.py – הדרך שלך ללמוד פייתון](https://campus.gov.il/course/course-v1-cs-gov_cs_selfpy101/) (קמפוסIL והמרכז לחינוך סייבר)
+* [Next.py – הצעד הבא שלך בפייתון](https://campus.gov.il/course/course-v1-cs-gov-cs-nextpy102/) (קמפוסIL והמרכז לחינוך סייבר)
+* [network.py לתכנת במרחב הרשת](https://campus.gov.il/course/cs-gov-cs-networkpy103-2020-1/) (קמפוסIL והמרכז לחינוך סייבר)
+
+
+### R
```
Comment (LuigiImVector) [inline courses/free-courses-he.md:18]:
```suggestion
### R

```

File: courses/free-courses-he.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+* [מבוא לתכנות ועיבוד נתונים בשפת R](https://campus.gov.il/course/telhai-acd-rfp4-telhai-r/) (קמפוסIL ומכללת תל־חי)
```
Comment (LuigiImVector) [inline courses/free-courses-he.md:19]:
```suggestion
* [מבוא לתכנות ועיבוד נתונים בשפת R](https://campus.gov.il/course/telhai-acd-rfp4-telhai-r/) (קמפוסIL ומכללת תל־חי)

```

**Review by LuigiImVector — COMMENTED** [review]
There weren't the [correct number of black lines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#formatting).

File: courses/free-courses-he.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+### Index
+
+* [C++](C++)
```
Comment (davorpa) [inline courses/free-courses-he.md:3]: Fix TOC entry pointing to an alias

```suggestion
* [C++](#cpp)
```

File: courses/free-courses-he.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+* [Python](#python)
+* [R](#R)
```
Comment (davorpa) [inline courses/free-courses-he.md:5]: Like in other listings, anchor goes in lowecase

```suggestion
* [R](#r)
```

File: courses/free-courses-he.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+
+### C++
```
Comment (davorpa) [inline courses/free-courses-he.md:8]: Add TOC alias
```suggestion
### <a name="cpp"></a>C++
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Thanks @tomer. Having a new localized listing is always a nice feature.

Please, solve minor bugs about TOC.

**Review by davorpa — APPROVED** [review]
Ready to merge :tada:

**Comment by eshellman** [comment]
love it. 
Thanks, and welcome to free-programming-books!

**Pull request merged**